app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_name, old_file, path = command.split()
```

Comment:
You could consider adding `*_` to collect excessive items in the command after split
```suggestion
    command_name, old_file, path, *_ = command.split()
```